template-processors/base/bin/resourceManager.sh

```diff
@@ -53,15 +53,20 @@ function createUpdateResources {
 
 }
 
-echo Managing Resources
+if [ $CREATE_MODE == "None" ] || [ $DELETE_MODE == "None" ]; then
+  echo "CREATE_MODE is set to None; This means that the template processor already applied the resources. Skipping the Manage Resources step."
```

Comment:
```suggestion
  echo "CREATE_MODE and/or DELETE_MODE is set to None; This means that the template processor manages the resources directly. Skipping the Manage Resources step."
```

Reply:
@etsauer your force push wiped this change out. I assume that was by accident?

Reply:
sorry, put it back.